doc/rst/technotes/gpu.rst

```diff
@@ -136,39 +139,125 @@ GPU-Related Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To enable GPU support set the environment variable ``CHPL_LOCALE_MODEL=gpu``
-before building Chapel.
+before building Chapel. Several other variables affect how Chapel generates
+code for and interacts with GPUs. These variables include:
+
+* ``CHPL_GPU`` --- may be set to ``nvidia``, ``amd', or ``cpu``. If unset, as
+  part of its build process, Chapel will attempt to automatically determine
+  what type of GPU you're trying to target. Changing this variable requires
+  rebuilding Chapel. For more information, see the `Vendor Portability`_
```

Comment:
All of Chapel? I thought it was just the runtime, and it may not even require a rebuild. I mean, I thought Engin closed that issue I opened...

https://github.com/chapel-lang/chapel/issues/21978

Reply:
Great point. Reworded to "may require rebuilding the Chapel runtime" ("may" since if you switch back-and-forth I don't think it would require rebuilding again).